Core/src/org/sleuthkit/autopsy/commonfilesearch/CommonFilesSearchNode.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Autopsy Forensic Browser
+ *
+ * Copyright 2018 Basis Technology Corp.
+ * Contact: carrier <at> sleuthkit <dot> org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.autopsy.commonfilesearch;
+
+import org.openide.nodes.AbstractNode;
+import org.openide.nodes.Children;
+import org.openide.util.NbBundle;
+import org.openide.util.lookup.Lookups;
+
+/**
+ * Encapsulates data used to display common files search results in the top
+ * right pane.
+ */
+final class CommonFilesSearchNode extends AbstractNode {
+
+    private final Children children;
+
+    CommonFilesSearchNode(CommonFilesMetaData metaData) {
+        super(Children.create(new CommonFilesChildren(metaData), true), Lookups.singleton(metaData));
+        children = this.getChildren();
```

Comment:
Is this.getChildren() the best call now that we've implemented the child factory?

Reply:
We don't need it.